controllers/OneCampaignMonitor_SubscribersController.php

```diff
@@ -8,6 +8,17 @@ class Onecampaignmonitor_SubscribersController extends BaseController
     public function actionAdd() {
         $this->requirePostRequest();
 
+        // Verify the recaptcha response before creating the user
+        if (craft()->plugins->getPlugin('OneCampaignMonitor')->getSettings()->verify_recaptcha) {
+            $response = craft()->request->getParam('recaptchaResult');
```

Comment:
can we change this to the default `g-recaptcha-response`? Otherwise you have to rename the default param which may not be obvious to developers down the road.

Reply:
Done!